ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -11231,6 +11231,40 @@ keys:
           - str
           min: 0
           max: 4294967295
+  stun:
+    type: dict
+    keys:
+      client:
+        type: dict
+        description: '"If a device is a client"
```

Comment:
@burnyd Please remove the extra quotes in the descriptions. Looks like we are referencing a quote in the rendered documentation